tests/libkvconfig/netbinding.go

```diff
@@ -74,3 +84,21 @@ func networkBindingPatch(name string, binding v1.InterfaceBindingPlugin) patch.P
 		Value: binding,
 	}
 }
+
+func RegisterPasstPlugin() error {
+	tests.EnableFeatureGate(virtconfig.NetworkBindingPlugingsGate)
```

Comment:
I think feature enablement should be visible at tests who use it instead of hiding in helpers.
By having it visible its easier to map and realize which test uses a feature gate.

Reply:
Explicit is better than implicit in this case.
Having a side effect of enabling a FG due to some other operation is questionable.

Reply:
Re-examining this, I think `RegisterPasstPlugin` should not be placed here. It can be places where it is needed directly.